pkg/specgen/generate/container.go

```diff
@@ -203,6 +205,17 @@ func CompleteSpec(ctx context.Context, r *libpod.Runtime, s *specgen.SpecGenerat
 	if len(s.LogConfiguration.Driver) < 1 {
 		s.LogConfiguration.Driver = rtc.Containers.LogDriver
 	}
+	if len(rtc.Containers.LogTag) > 0 {
+		if s.LogConfiguration.Driver != define.JSONLogging {
+			if s.LogConfiguration.Options == nil {
+				s.LogConfiguration.Options = make(map[string]string)
+			}
+
+			s.LogConfiguration.Options["tag"] = rtc.Containers.LogTag
+		} else {
+			logrus.Warnf("log_tag %q is not allowed with %q log driver", rtc.Containers.LogTag, define.JSONLogging)
```

Comment:
Since the field name is `log_driver`, just a nit:
```suggestion
			logrus.Warnf("log_tag %q is not allowed with %q log_driver", rtc.Containers.LogTag, define.JSONLogging)
```